Google.PowerShell/Compute/GceCmdlet.cs

```diff
@@ -132,7 +132,7 @@ private void WriteWarnings(Operation op)
         private void WriteProgress(Operation op, string progressMessage)
         {
 
-            int activityId = op.Id.GetHashCode();
+            int activityId = Math.Abs(op.Id.GetHashCode());
```

Comment:
This seems odd to me.  Is this common practice?  If not a comment would be nice (and below)